### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Sebbybobbler**

### CODE REVIEW

File: src/components/Header/Header.module.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    flex-direction: column;
+    justify-content: end;
+    z-index: -1;
+    background-color: white;
+    overflow:hidden;
+    min-height:1000px;
+    min-width:1100px;
+    border-radius: 100% 100% 100% 100%;
+    background-position:center;
+    margin-top: -60%;
```
Comment (Sebbybobbler) [inline src/components/Header/Header.module.css]:
```suggestion
    margin-top: -50%;
```

File: public/images/ducklogo.png
Comment (ChloeSAPage) [inline public/images/ducklogo.png:1]: ?

**Review by ChloeSAPage — APPROVED** [review]

**Comment by Sebbybobbler** [comment]
image

File: src/components/Header/Header.module.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    flex-direction: column;
+    justify-content: end;
+    z-index: -1;
+    background-color: white;
+    overflow:hidden;
+    min-height:1000px;
+    min-width:1100px;
+    border-radius: 100% 100% 100% 100%;
+    background-position:center;
+    margin-top: -50%;
```
Comment (Sebbybobbler) [inline src/components/Header/Header.module.css]:
```suggestion
    margin-top: -55%;
```

**Comment by ChloeSAPage** [comment]
> image

beautiful

**Comment by NicoClack** [comment]
Looks good. If this was a longer project I'd nag you to run Prettier on it and remove the curly brackets around the image srcs though.

**Review by NicoClack — APPROVED** [review]

**Review by BlueElyas — APPROVED** [review]

**Pull request merged**